app/src/editor/canvas/components/ModuleSearch.pcss

```diff
@@ -63,18 +62,31 @@
   color: #A3A3A3;
 }
 
-.ModuleSearch .Input input {
+.ModuleSearch .Header svg:hover path {
+  fill: #525252;
+  stroke: #525252;
+}
+
+.ModuleSearch .Input {
```

Comment:
Can we go with a better alignment?

<img width="111" alt="Screenshot 2019-05-03 10 04 11" src="https://user-images.githubusercontent.com/320066/57125771-e23be000-6d8a-11e9-9bcb-c87d0b922887.png">

Reply:
Was going to do this but the icons in the header need changing anyway